### ISSUE TITLE
Flutter web fails to connect using SockJs

### ISSUE THREAD

**Issue by Petrol-** [blackhorse-one/stomp_dart_client#32]
Hello,

I'm trying to use the SockJs implementation with flutter for iOS, Android and Web.

It works great with Android and iOS but it fails to connect with flutter web with the following error in the last catch of StompHandler.connect() : 

`DomException (SyntaxError: Failed to construct 'WebSocket': The URL contains a fragment identifier (''). Fragment identifiers are not allowed in WebSocket URLs.)`

It seems that the problem is SockJsUtils.generateTransportUrl(String url) which generates **transportUrl** with a trailing '#'

I've written a simple test that shows the problem:

```
test('generate websocket url with no empty fragment', () {
      final url = 'https://localhost:5000/test';

      final webSocketUrl = SockJsUtils().generateTransportUrl(url);

      expect(webSocketUrl, isNotNull);
      expect(webSocketUrl, isNotEmpty);
      expect(webSocketUrl.endsWith('#'), isFalse);
    });
```

**Comment by KammererTob** [comment]
Sorry for the late reply and thanks for your work on this! I will check your PR very soon.

**Comment by KammererTob** [comment]
Closed by merge of #33
